src/backend/controllers/HistoryController.js

```diff
@@ -89,12 +89,12 @@ export const removeVideoFromHistoryHandler = function (schema, request) {
         404,
         {},
         {
-          errors: ["The email you entered is not Registered. Not Found error"],
+          errors: ['The email you entered is not Registered. Not Found error'],
         }
       );
     }
     const videoId = request.params.videoId;
-    const filteredHistory = user.history.filter((item) => item._id !== videoId);
+    const filteredHistory = user.history.filter((item) => item._id != videoId);
```

Comment:
Why are you not using strict equality?

Reply:
the item._id is of type number and the videoId is of type string, so if I use strict equality it will return false even if their values are the same, so one solution is to convert them to number or string and perform strict equality, and another option is it not check their types at all, because all we care about is their value and not their types, so therefore not using the strict equality here.